paper-dropdown-menu.html

```diff
@@ -110,16 +110,17 @@
     }
 
     paper-ripple {
-      top: 20px;
```

Comment:
Hmm, it looks almost the same. You're right, there's some bonus pixels, so i think this needs to be 20-8=12 instead of 8.

Before the PR
<img width="272" alt="screen shot 2015-10-07 at 10 21 57 am" src="https://cloud.githubusercontent.com/assets/1369170/10345277/5d4c6c72-6cdd-11e5-9aaa-1ac00ff16906.png">

With this PR:
<img width="268" alt="screen shot 2015-10-07 at 10 22 01 am" src="https://cloud.githubusercontent.com/assets/1369170/10345281/61cb9890-6cdd-11e5-94b5-e07bacd55c2c.png">